### DISCUSSION TITLE
Is it possible to use DatePicker and Combobox in uncontrolled form

### DISCUSSION THREAD

**Discussion by dmitry-ff** [mantinedev/mantine#6961] [Q&A] [answered]
I can’t understand whether it’s possible to use DatePicker and Combobox inside a form with the uncontrolled mode, because when the value in these fields changes, the ui is not updated

**Comment by rtivital** [comment, marked as answer]
DatePicker and Combobox components are not inputs – they are components to build inputs. The logic is the same as for dropzone – https://help.mantine.dev/q/how-to-use-dropzone-with-form
